libs/vue-plugin/src/schematics/application/schema.json

```diff
@@ -23,6 +23,12 @@
       "description": "A directory where the project is placed",
       "alias": "d"
     },
+    "style": {
```

Comment:
Consider x-prompt.

Reply:
![DO IT!](https://media.giphy.com/media/qDPg6HNz2NfAk/source.gif)